src/content/docs/style-guide/word-choice/usage-dictionary.mdx

```diff
@@ -466,6 +491,25 @@ For consistency across New Relic content, here are our general word usage guidel
     The list of [`pages`](#pages) and [`indexes`](#index) on the top and left sides of the New Relic user interface.
   </Collapser>
 
+
+  <Collapser
+    id="metric"
+    title="metric"
+  >
+For an explanation of how the word "metric" is used in the industry and how it's used at New Relic, see [Data types](/docs/data-apis/understand-data/new-relic-data-types/#metrics). 
+
+Some tips about some often confusing aspects of metric data: 
```

Comment:
great section!! i like that it's clear on what our audiences need to know, but still provides that info to build our mental map/give more context. I like the little trick of just linking to the timeslice data if we think it could be relevant too.